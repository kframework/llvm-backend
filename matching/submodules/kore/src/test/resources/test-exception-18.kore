[]

module TEST-EXCEPTION-18

  axiom{}
    \ceil{#S,S}(X:S) []

endmodule []
