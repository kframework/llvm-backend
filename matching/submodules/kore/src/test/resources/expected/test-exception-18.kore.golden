Parse error: ../../../test/resources//test-exception-18.kore:6:14:
unexpected 'S'
expecting '#'
